File: Leetcode/weekly contest - 277/B.js
var rearrangeArray = function (nums) {
  let p = [],
    n = [];
  nums.map((a) => (0 < a ? p.push(a) : n.push(a)));

  let ans = [];
  p.forEach((item, index) => {
    ans.push(item);
    ans.push(n[index]);
  });

  return ans;
};

File: Leetcode/weekly contest - 277/B.py
class Solution:
    def rearrangeArray(self, nums: List[int]) -> List[int]:
        n,p = [], []
        for item in nums:
            if item > 0 :
                p.append(item)
            else:
                n.append(item)
        
        res = []
        for i in range(0,len(p)):
            res.append(p[i])
            res.append(n[i])
            
        return res